Documentation/ReleaseNotes.md

```diff
@@ -39,6 +39,26 @@ A [Leap Motion Controller](https://www.ultraleap.com/product/leap-motion-control
 
 ![LeapMotionIntroGif](Images/CrossPlatform/LeapMotion/LeapMotionSideBySide2.gif)
 
+**Migration window**
+
+![Migration window](Images/MigrationWindow/MRTK_Migration_Window.png)
+
+MRTK now comes with a migration tool that will help you upgrade deprecated components to their newer
+versions and to keep existing code working even as MRTK makes breaking changes.
+
+**It is generally recommended to run the migration tool after pulling a new version of MRTK** to
```

Comment:
Should you mention that the migration Window is in the Tools package?

Reply:
It's mentioned on line 53